<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>90</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);

</string>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>91</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 100);
font:  &quot;微软雅黑&quot;;</string>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
    <item>
     <property name="text">
      <string>数学</string>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>物理</string>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>化学</string>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>生物</string>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>历史</string>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>英语</string>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>政治</string>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>地理</string>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="flags">
      <set>ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>语文</string>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_CurrentTime">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 17pt &quot;微软雅黑&quot;;
background-color: rgba(0, 0, 0, 100);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>19:20</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 100);</string>
    </property>
    <property name="text">
     <string>۞</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
